numalogic/connectors/druid.py

```diff
@@ -100,14 +104,21 @@ def fetch(
         dimensions: list[str],
         granularity: str = "minute",
         aggregations: Optional[dict] = None,
+        post_aggregations: Optional[dict] = None,
         group_by: Optional[list[str]] = None,
-        pivot: Optional[Pivot] = None,
+        pivot: Optional[Pivot] = Pivot(),
```

Comment:
Any reason why we are initializing here?

Reply:
Otherwise pivot.columns throws an error in the method

Reply:
But we can't use a mutable type in the function arts. Need to find a different way here.